src/java/org/apache/cassandra/batchlog/BatchlogManager.java

```diff
@@ -136,10 +137,11 @@ public static void remove(UUID id)
 
     public static void store(Batch batch)
     {
-        store(batch, true);
+        // WATCH OUT: here true was changed to null (the same was done in DSE)
+        store(batch, WriteOptions.DEFAULT);
```

Comment:
is that change fine?

Reply:
I think it's fine, `System` keyspace will use `KeyspaceParams#DEFAULT_LOCAL_DURABLE_WRITES = true` to determine whether to write to commitlog.

It's better to remove the reference to DSE and update the comment to mention "DEFAULT_LOCAL_DURABLE_WRITES"